lbcapi/api.py

```diff
@@ -32,7 +35,11 @@ def __init__(self):
         self.hmac_key = None
         self.hmac_secret = None
 
-    def call(self, method, url, params={}, stream=False, files=None):
+    def call(self, method, url, params=None, stream=False, files=None):
```

Comment:
https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python